src/lib.rs

```diff
@@ -155,6 +155,9 @@ compile_error!("A least one underlying storage type must be enabled. See the fea
 
 #[doc(hidden)]
 pub extern crate num;
+#[doc(hidden)]
```

Comment:
I think this should be `#[cfg_attr(feature = "serde", doc(hidden)]` so that the attribute only appears when `serde` is enabled. Didn't test with the compiler however.

Reply:
I tested running `cargo doc` with and without the `serde` feature enabled and it seemed to work fine in both cases (not showing docs for serde in either case). However, of course, if you'd like me to specify a cfg_attr anyway I can do that.